client/my-sites/post-share/style.scss

```diff
@@ -1,177 +1,278 @@
 $section-border: solid 1px transparentize( lighten( $gray, 20% ), .5 );
 
-.posts__list .post-share__wrapper {
-	margin: 0;
-	animation: appear .3s ease-in-out;
+.post-share {
 	border-top: $section-border;
 
-	.post-share__title {
-		margin: 24px 0 0 0;
-		font-size: 18px;
-	}
+	.post-share__service.is-active,
+	.post-share__handle {
+		.social-logo.twitter{
+			color: $color-twitter;
+		}
+
+		.social-logo.facebook{
+			color: $color-facebook;
+		}
+
+		.social-logo.google-plus {
+			color: $color-gplus;
+		}
 
-	.post-share__subtitle {
-		margin-bottom: 24px;
+		.social-logo.linkedin {
+			color: $color-linkedin;
+		}
+
+		.social-logo.tumblr {
+			color: $color-tumblr;
+		}
+
+		.social-logo.path {
+			color: $color-path;
+		}
+
+		.social-logo.eventbrite {
+			color: $color-eventbrite;
+		}
 	}
 
-	.post-share__main {
-		border-top: $section-border;
-		display: flex;
-		flex-direction: column-reverse;
+	.card.banner {
+		margin: 16px;
 
 		@include breakpoint( ">480px" ) {
-			flex-direction: row;
+			margin: 16px 24px;
 		}
 	}
+}
 
-	.post-share__form.is-placeholder,
-	.post-share__services.is-placeholder {
-		@include placeholder;
-		margin-top: 16px;
-		height: 48px;
-	}
+.post-share__wrapper {
+	animation: appear .3s ease-in-out;
+	margin: 0;
 
-	.post-share__head {
-		padding: 0 24px;
+	.notice {
```

Comment:
Instead of writing this inside `.post-share__wrapper`, I believe our convention is to write in a top level:

```
.post-share__wrapper {
...
}

.post-share__wrapper.notice {
...
}
```